lib/chat_models/ollama_ai_fields.ex

```diff
@@ -0,0 +1,78 @@
+defmodule LangChain.ChatModels.OllamaAIFields do
+  defmacro fields do
+    quote do
+      @receive_timeout 60_000 * 5
+
+      field :endpoint, :string, default: "http://localhost:11434/api/chat"
```

Comment:
Is there any support for overriding this endpoint? This default value should be explained in the ChatOllamaAI's module doc along with information or an example of how to override.

Reply:
Will make a note in the module docs on this.  The app within the [image exposes port 11434](https://github.com/jmorganca/ollama/blob/main/Dockerfile#L119) but their is a world where a client may want to deploy the image with a different port mapping thus requiring an override